esmvaltool/recipes/recipe_climwip_test_basic.yml

```diff
@@ -41,11 +47,34 @@ preprocessors:
       scheme: linear
     mask_landsea:
       mask_out: sea
-    extract_region:
-      start_longitude: -10.0
-      end_longitude: 39.0
-      start_latitude: 30.0
-      end_latitude: 76.25
+
+    # -- cut region using corners --
+    # extract_region:
+    #   start_longitude: -10.0
+    #   end_longitude: 39.0
+    #   start_latitude: 30.0
+    #   end_latitude: 76.25
+
+    # -- cut SREX regions --
+    # extract_shape:
+    #   shapefile: shapefiles/srex.shp
+    #   method: contains
+    #   crop: true
+    #   ids:
+    #     - 'North Europe [NEU:11]'
+    #     - 'Central Europe [CEU:12]'
+    #     - 'South Europe/Mediterranean [MED:13]'
```

Comment:
what is the purpose of commented-out lines? If it's a test case or any other corner case, it's good to explicitly write what it is for at the top of the commented-out block

Reply:
It's two other ways of doing the same thing. For the European regions SREX and AR6 are identical but for other regions this is not the case, and they have updated names.